scripts/vagrantCloudBasedContainers.sh

```diff
@@ -15,44 +15,85 @@
 #
 # https://issues.gpii.net/browse/GPII-2470
 
+# This script accepts an option "--no-rebuild" indicating to only start
+# docker containers without rebuilding the universal docker image. If this
+# option is not provided, the old docker containers will be removed and the
+# universal docker image will be rebuilt.
+# Example command to build from the scratch: ./vagrantCloudBasedContainers.sh
+# Example command not to rebuild: ./vagrantCloudBasedContainers.sh --no-rebuild
+
+if  [ "$1" == "--no-rebuild" ]; then
+    NO_REBUILD=true
+fi
+
 UNIVERSAL_IMAGE=vagrant-universal
 
 COUCHDB_IMAGE=couchdb
 COUCHDB_PORT=5984
 COUCHDB_HEALTHCHECK_DELAY=2
 COUCHDB_HEALTHCHECK_TIMEOUT=30
 
-DATALOADER_IMAGE="gpii/preferences-dataloader"
-DATALOADER_CLEAR_INDEX=1
-DATALOADER_COUCHDB_URL="http://couchdb:${COUCHDB_PORT}/preferences"
+STATIC_DATA_DIR="/home/vagrant/sync/universal/testData/dbData"
+BUILD_DATA_DIR="/home/vagrant/sync/universal/build/dbData"
+
+DATALOADER_IMAGE="cindyqili/gpii-dataloader"
+DATALOADER_COUCHDB_URL="http://couchdb:${COUCHDB_PORT}/gpii"
+DATASOURCE_HOSTNAME="http://couchdb"
 
 PREFERENCES_CONFIG="gpii.config.preferencesServer.standalone.production"
-PREFERENCES_DATASOURCE_URL="http://couchdb:${COUCHDB_PORT}/preferences/%userToken"
-PREFERENCES_DIR="/home/vagrant/sync/universal/testData/preferences"
 PREFERENCES_PORT=9081
 
 FLOWMANAGER_CONFIG="gpii.config.cloudBased.flowManager.production"
-FLOWMANAGER_PREFERENCES_URL="http://preferences:${PREFERENCES_PORT}/preferences/%userToken"
 FLOWMANAGER_PORT=9082
+FLOWMANAGER_MATCHMAKER_URL="http://localhost:9082"
```

Comment:
I know we didn't do this before, but would it be worth prefixing all of these env variables with GPII_?